woocommerce/woocommerce
# [Accessibility] Add aria attributes for comments reply span title

### Submission Review Guidelines:


### Changes proposed in this Pull Request:

Closes #51905 .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Browse to a product page on the frontend with no reviews.
2. Ensure that reviews are enabled before proceeding.
3. Browse to the product reviews tab.
4. Ensure "Be the first to review ..." is an `<span>` tag with role="heading" aria-level="3".
5. Leave a comment
6. Ensure "Add a review" is an `<span>` tag with role="heading" aria-level="3".

Test Accessibility in using Chrome dev tools.

1. Open Dev Tools
2. Select "Accessibility" under "Elements"
3. Enable "Enable full-page accessibility tree"
4. Reload the page
5. Ensure the heading "Be the first to review ..." or "Add a review" are present in the Accessibility Tree.

### Changelog entry


<details>

<summary>Changelog Entry Details</summary>

#### Significance


#### Type


Replace span with h3 for comments reply title.

</details>
